reframe/core/pipeline.py

```diff
@@ -708,6 +708,14 @@ def disable_hook(cls, hook_name):
     extra_resources = fields.TypedField('extra_resources',
                                         typ.Dict[str, typ.Dict[str, object]])
 
+    #: .. versionadded:: 3.3
```

Comment:
This goes at the end.

Reply:
Are you sure? Because in the filelds `max_pending_time`, `postrun_cmds`, `prerun_cmds` and all the others that were added at some point, it's the first thing. Should I change this everywhere?

Reply:
🤔  I thought it was the other way around. Let me check.

Reply:
You are right. Nonetheless we will need to expand the documentation of this field. For example, what does it mean when this is set to `False`